Server/index.js

```
@@ -4,6 +4,57 @@ const { PrismaClient } = require('@prisma/client');
+          message:
+            'Please indicate if you are joining us for a walk the day after.',
+        }),
+      })
+      .transform((val) => val === 'true')
+      .or(z.literal('')),
+  })
+  .superRefine((data, ctx) => {
+    if (data.isAttending) {
+      if (
+        data.numberOfGuests === undefined ||
+        isNaN(data.numberOfGuests) ||
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)